src/stampy_nlp/faq_titles.py

```diff
@@ -82,11 +90,16 @@ def encode_faq_titles():
 
 
 if __name__ == "__main__":
+    from stampy_nlp.models import connect_pinecone
+    from stampy_nlp.utilities.pinecone_utils import get_index
+
     logging.basicConfig(level=logging.DEBUG)
 
-    if len(sys.argv) > 1 and sys.argv[1] == '-delete-all':
-        logger.debug("sys.argv[1]=%s", sys.argv[1])
+    connect_pinecone(get_index())
+
+    if any("-delete-all" in s for s in sys.argv):
         delete_all = True
-    logger.debug("delete_all=%s", delete_all)
+        # logger.isEnabledFor("sys.argv[1]=%s", sys.argv[1])
+    logger.info("delete_all=%s", delete_all)
```

Comment:
What does `logger.isEnabledFor("sys.argv[1]=%s", sys.argv[1])` do? I'm patting parameter `-delete-all` on the command line to clear out the pinecone db periodically which should enable or disable logger.

Reply:
no idea :D